web/srv/api_handlers.go

```diff
@@ -438,3 +438,31 @@ func (h *handler) handleAPIResourceDefinition(w http.ResponseWriter, req *http.R
 	w.Header().Set("Content-Type", "text/yaml")
 	w.Write(resourceDefinition)
 }
+
+func (h *handler) handleAPIGateways(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
+	requestParams := util.GatewayRequestParams{
```

Comment:
Thanks for updating and removing the helper function!  However, I was getting at is that we can remove this params structure entirely and just construct the request directly.  Sorry if that wasn't clear.  What I had in mind was something like this:

```
	window := req.FormValue("window")
        if window == "" {
                window = "1m"
        }
	_, err := time.ParseDuration(window)
	if err != nil {
 		renderJSONError(w, err, http.StatusInternalServerError)
		return
	}
	gatewayRequest := &pb.GatewaysRequest{
		TimeWindow:        window,
		GatewayNamespace:  req.FormValue("gatewayNamespace"),
		RemoteClusterName: rreq.FormValue("remoteClusterName"),
	}
```